src/services/admin.ts

```diff
@@ -0,0 +1,13 @@
+import Axios from './axios';
+
+export const isAdmin = async (email: string) => {
```

Comment:
Do we need types on functions like this?

Reply:
We'll talk later, but i think the backend is the one who need to declare the type at the first place.